### PR TITLE
APP-1339 - Fix QA comments

### PR DESCRIPTION
**Pull request by luanapp**

### CODE REVIEW

File: src/main/webapp/services/assignUserService.js [hunk 1]
```diff
@@ -131,8 +133,6 @@ export default class AssignUserService extends BaseService {
         this.successDialog(data);
       })
       .catch((error) => {
-        this.selectedUser = {};
```
Comment (robson-sanchez) [inline src/main/webapp/services/assignUserService.js:134]: Why did you remove this statement? We should clean up old values when you render again the component.

File: src/main/webapp/services/assignUserService.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ export default class AssignUserService extends BaseService {
     };
 
     this.updateDialog('assignIssue', template, userData);
+    setTimeout(() => this.closeDialog('assignIssue'), 3000);
```
Comment (robson-sanchez) [inline src/main/webapp/services/assignUserService.js:37]: Is there any indication for the user that will happen? I don't think the user is expecting this behavior.

File: src/main/webapp/services/commentService.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ export default class CommentService extends BaseService {
 
     const template = dialogBuilder.build(data);
     this.updateDialog('commentIssue', template, {});
+    setTimeout(() => this.closeDialog('commentIssue'), 3000);
```
Comment (robson-sanchez) [inline src/main/webapp/services/commentService.js:27]: The same comment above.

**Review by robson-sanchez — CHANGES_REQUESTED** [review]

File: src/main/webapp/services/assignUserService.js (hunk 2 above)
Comment (luanapp) [reply src/main/webapp/services/assignUserService.js:37]: The last dialog after assigning the issue is a dead end, as the user can't do anything besides closing the dialog. This is a suggestion from Vince, so we can ask the UX team what should be the best behavior.

File: src/main/webapp/services/assignUserService.js (hunk 1 above)
Comment (luanapp) [reply src/main/webapp/services/assignUserService.js:134]: So, we shouldn't. As the UX proposal says, the error should appear and the selected user must remain. See comment number 2 in the issue

**Review by robson-sanchez — APPROVED** [review]
LGTM
